### PR TITLE
Added Logger

### LINKED ISSUE #18: Adding Logger
Add logging feature that keeps record of status of the API.

### PR DESCRIPTION
**Pull request by karelumair**
added logger to keep logs of activities.

closes #18

### CODE REVIEW

**Review by pratikGhodke1 — CHANGES_REQUESTED** [review]
1. Update logger to log custom messages
2. Show logs on console as well as save that to files

File: app.py [hunk 1]
```diff
@@ -4,15 +4,14 @@
...
 import os
 
 load_dotenv(find_dotenv())
 
-# Logging Configuration
-logging.basicConfig(filename='logs.log',
-                level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(name)s - %(message)s')
-
+logger = logging.getLogger("app")
+init_logger(logger, os.environ.get("LOG_FILE"))
```
Comment (pratikGhodke1) [inline app.py]: You can avoid passing LOG FILE as param here and use the constant directly in logger.py as it is an ENV variable

**Review by pratikGhodke1 — CHANGES_REQUESTED** [review]
Make a small change in passing parameter in init_logger function as mentioned in other conversion.

File: app.py (hunk 1 above)
Comment (karelumair) [reply app.py]: sure

**Pull request merged**